apiml-static-registration.yaml.template

```diff
@@ -0,0 +1,11 @@
+services:
+  - serviceId: explorer-jes
+    title: JES Explorer
+    description: IBM z/OS Jobs UI service
+    catalogUiTileId:
+    instanceBaseUrls:
+      - https://${ZOWE_EXPLORER_HOST}:${JES_EXPLORER_UI_PORT}/
+    homePageRelativeUrl:
+    routedServices:
+      - gatewayUrl: ui/v1
+        serviceRelativeUrl: ${EXPLORER_PLUGIN_BASEURI}
```

Comment:
We need to make sure this variable has a value with current instance.env when we want to parse the template with new component launcher script. Or pick value from `configs.baseUri`.

Reply:
Yep - I've updated it and added a note to indicate that when we start using template injection we need to get this value from the parsed component config or directly from manifest.yaml